tasks/lzards-backup/src/index.ts

```diff
@@ -268,14 +278,31 @@ export const backupGranule = async (params: {
     cloudfrontEndpoint?: string,
   },
 }) => {
+  let granuleCollection : CollectionRecord;
+  let collectionId: string = '';
   const { granule, backupConfig } = params;
+  const messageGranule = granule as MessageGranuleFromStepOutput;
+  const apiGranule = granule as ApiGranule;
   log.info(`${granule.granuleId}: Backup called on granule: ${JSON.stringify(granule)}`);
   try {
-    const granuleCollection = await getGranuleCollection({
-      collectionName: granule.dataType,
-      collectionVersion: granule.version,
-    });
-    const collectionId = constructCollectionId(granule.dataType, granule.version);
+    if (apiGranule.collectionId) {
+      collectionId = apiGranule.collectionId;
+      const { name, version } = deconstructCollectionId(apiGranule.collectionId);
+      granuleCollection = await getGranuleCollection({
+        collectionName: name,
+        collectionVersion: version,
+      });
```

Comment:
We might be able to DRY this up a bit if ya like. Maybe something like 

```
let name;
let version;

if (apiGranule.collectionId) {
  {name, version} = deconstructCollectionId(apiGranule.collectionId);
} else if (messageGranule.dataType && messageGranule.version) {
  name = messageGranule.dataType;
  version = messageGranule.version
}

granuleCollection = await getGranuleCollection({
        collectionName: name,
        collectionVersion: version,
      });

collectionId = constructCollectionId(name, version);
```

Reply:
Yeah, good call out! I mostly took the suggestion above but modified it a bit since I don't think this line is possible `  {name, version} = deconstructCollectionId(apiGranule.collectionId);`
so I instead did
```
      const collectionNameAndVersion = deconstructCollectionId(apiGranule.collectionId);
      name = collectionNameAndVersion.name;
      version = collectionNameAndVersion.version;
```
https://github.com/nasa/cumulus/pull/3043/commits/ad477d13003920919959a9deef5c8793b2e4cd09#diff-31ab5953f1d25d2a7e338fb25acc5c7ae53aacc8cde330d0084eb4e00236cf3fR283

Reply:
Yup, that looks great. Thanks!